data/core/encyclopedia/geography.cfg

```diff
@@ -138,7 +138,7 @@ Over the River Aethen, south of Fort Tahn, is a Wesnothian frontier region. It i
 [topic]
     id=northlands
     title= _ "Northlands"
-    text= _ "There is no government of the Northlands. Various groups of orcs, dwarves, barbarian men and even elves populate the region. The northern and eastern borders are not defined, the southern border is the <ref>dst='great_river' text='Great River'</ref>, and the western border is the <ref>dst='great_ocean' text='ocean'</ref>. The mostly subterranean dwarvish kingdoms of Knalga and Kak Kartha are found here.
+    text= _ "There is no government over the freezing Northlands, only unstable clans in endless conflict. It is an unlawful and perilous place plagued by both its cold, icy terrain and the barbarians inhabiting it. Various groups of orcs, dwarves, men, gryphons, ogres, and even elves populate the region. Yet despite the sheer wilderness above the far north, the subterranean Dwarvish Kingdoms of Knagla and Kal Kartha are exceptions to their anarchic surroudings, providing both heat and protection to their people.While they may often be in conflict with orcs and trolls, the heart of Dwarvish territory is considered relatively safe in comparison. It is ruled by a functioning and usually, stable government. The northern and eastern borders are not defined, the southern border is the <ref>dst='great_river' text='Great River'</ref>, and the western border is the <ref>dst='great_ocean' text='ocean'</ref>.
```

Comment:
Can we not use the term "plagued" to refer to people? I think something like "plagued by its cold, icy terrain and inhabited chiefly by savage barbarians" might work. Not entirely happy with "savage barbarians" either, but it's better than "plagued" in my opinion.

I'm also unhappy with the use of "anarchic", though it's technically correct; I just don't like seeing it used in this sense, as it's not necessarily synonymous with "chaotic". That said, I'm okay with it staying if no-one has better wording to suggest. (You're missing a space after that sentence, by the way.)

The comma doesn't belong in "functioning and usually stable government". Though, it would work if there were also a comma _before_ "usually"… but personally I think it flows better without commas.